### PR TITLE
Fix schedule

### PR DESCRIPTION
**Pull request by matthewbentley**
This works, but I want to wait to make sure the docker hub builds work before merging https://hub.docker.com/r/hacsoc/jolly-advisor-cron/builds/

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,21 +2,12 @@ FROM ruby:2.2.3-onbuild
 
 MAINTAINER Matthew Bentley <matthew.t.bentley@gmail.com>
 
-RUN apt-get update && \
-    apt-get install -y \
-      vim \
```
Comment (ajm188) [inline Dockerfile]: why are we ditching vim?

File: cron/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+FROM hacsoc/jolly-advisor
+
+MAINTAINER Matthew Bentley <matthew.t.bentley@gmail.com>
+
+RUN apt-get update && apt-get install -y \
+    postgresql-client cron \
```
Comment (ajm188) [inline cron/Dockerfile]: don't we already have postgresql-client installed via hacsoc/jolly-advisor?

File: cron/crontab [hunk 3]
```diff
@@ -0,0 +1,3 @@
+30 4 * * * /bin/bash -l -c 'cd /usr/src/app && RAILS_ENV=production bundle exec rake import:data_dump' > /proc/1/fd/1
+
+* * * * * /bin/bash -l -c 'echo test5' > /proc/1/fd/1
```
Comment (ajm188) [inline cron/crontab]: looks like you forgot to remove this

File: cron/crontab [hunk 4]
```diff
@@ -0,0 +1,2 @@
+30 4 * * * /bin/bash -l -c 'cd /usr/src/app && RAILS_ENV=production bundle exec rake import:data_dump' > /proc/1/fd/1
+
```
Comment (ajm188) [inline cron/crontab]: extreme nit pick: can we remove the extra newline?

**Comment by ajm188** [comment]
:ship: looks good to me.

**Comment by matthewbentley** [comment]
Why did the most recent travis fail?

**Comment by matthewbentley** [comment]
Otherwise, it's working fine

**Comment by ajm188** [comment]
Because of #180

**Comment by matthewbentley** [comment]
Darn

**Pull request merged**